starter-code/lib/password.js

```diff
@@ -0,0 +1,38 @@
+var _ = require('lodash');
+
+var goodPsswd = "1234567890";
+var badPsswd = "1123456";
+
+// Different characters
+function noRepeatChar(password) {
+  var check = _.uniq(password);
+  if(check.length < password.length ){
+    console.log('error');
+  }
+  else{
+    console.log('guay');
+  }
+}
+
+noRepeatChar(badPsswd);
+
+// Only numbers
+function onlyNumbers(password){
+  _.parseInt(password);
+  for(var i=0; i<password.length; i++){
```

Comment:
Fijáos que estás recorriendo un bucle sin utilizar los índices en su interior. Tal y como lo tenéis, no necesitáis el for